hypothesis-python/tests/numpy/test_gen_data.py

```diff
@@ -655,10 +700,25 @@ def test_broadcastable_shape_has_good_default_values(shape, data):
     _broadcast_shapes(shape, broadcastable_shape)
 
 
+@settings(deadline=None, max_examples=200)
+@given(base_shape=ANY_SHAPE, num_shapes=st.integers(1, 10), data=st.data())
+def test_mutually_broadcastableshapes_has_good_default_values(
+    num_shapes, base_shape, data
+):
+    # This test ensures that default parameters can always produce broadcast-compatible shapes
+    shapes, result = data.draw(
+        nps.mutually_broadcastable_shapes(num_shapes=num_shapes, base_shape=base_shape),
+        label="shapes, result",
+    )
+    assert len(shapes) == num_shapes
+    # raises if shapes are not mutually-compatible
+    assert result == _broadcast_shapes(base_shape, *shapes)
+
+
 @settings(deadline=None)
 @given(min_dim=st.integers(0, 32), shape=ANY_SHAPE, data=st.data())
 def test_broadcastable_shape_can_broadcast(min_dim, shape, data):
-    max_dim = data.draw(st.none() | st.integers(min_dim, 32), label="max_dim")
+    max_dim = data.draw(st.none() | st.integers(min_dim, 32), label="max_dims")
```

Comment:
Here, and in the other test, you may want to rename `min_dim -> min_dims` too for consistency.